src/AppInstallerCLITests/TestCommon.cpp

```diff
@@ -32,6 +32,12 @@ namespace TestCommon
         static std::vector<std::filesystem::path> s_TempFilesOnFile;
 
         static std::filesystem::path s_TestDataFileBasePath{};
+
+        bool CleanVolatileTestRoot(HKEY root)

```

Comment:
>bool [](start = 8, length = 4)

Does not look like a need to return bool since we are always returning true and throw on failure?

Reply:
I saw what you're trying to do

---
In reply to: [519236814](https://github.com/microsoft/winget-cli/pull/633#discussion_r519236814) [](ancestors = 519236814)